base/linalg/transpose.jl

```diff
@@ -126,7 +126,6 @@ function ctranspose(A::AbstractMatrix)
     ctranspose!(B, A)
 end
 
-@inline ctranspose(A::AbstractVector{<:Real}) = transpose(A)
```

Comment:
Do I understand correctly that the `AbstractVector{<:Real}` part of the union in the former single `ctranspose(A::AbstractVecOrMat{<:Real})` definition (split into this pair of definitions in https://github.com/JuliaLang/julia/commit/2421ff2c150780e1a0c1835aab98acde69bd0e04) was effectively shadowed by the [more specific definition in rowvector.jl](https://github.com/JuliaLang/julia/blob/9706c8b67fe03dd859cbb01427ab0e618414ca46/base/linalg/rowvector.jl#L78)? And only after the split these definitions collide? If so, this change looks good to me. (Though perhaps the definition being removed is more fundamental than the definition in rowvector.jl, and that latter definition should go instead?) Best!